test/DebugInfo/NonSemantic/Shader200/DebugInfoSubrange.ll

```diff
@@ -97,10 +97,10 @@ attributes #1 = { nofree nosync nounwind readnone speculatable willreturn }
 !6 = !{i32 2, i32 0}
 !7 = !{}
 !8 = !{i16 6, i16 14}
-!9 = !{i32 0}
-!10 = !{!"none"}
-!11 = !{!"structtype"}
-!12 = !{!""}
+!9 = !{i32 0, i32 0, i32 0}
```

Comment:
Can we also add a test which checks for the error when invalid metadata is found?

Thanks

Reply:
Keep in mind that this is currently only checked with an assert; such a test wouldn't work when asserts are compiled out, it would have to be reworked to use `checkError` instead.  Not sure that's really worth it, just asserting that the input is valid (as done in the current patch) is fine with me.

Reply:
I agree with Sven on this one, I think just an assert is enough, let me know if you disagree.